config/routes.rb

```diff
@@ -71,6 +71,8 @@
         post :new
         post :start
       end
+
+      resource :submit_drafts, only: [:update]
```

Comment:
Toto mi nepride uplne railsy. Skor by som tu dal resource :drafts a nad tym post akciu `submit`

Reply:
Dala som to podla zvysnych endpointov. Chceme to cele upravit? @jsuchal https://github.com/slovensko-digital/govbox-pro/blob/e668f85524db8c73ff2f92864eafbd32bfa6d79e/config/routes.rb#L67-L68